src/features/reports/publisher_records/export_S21/specific_records/useParentUncheckHandler.tsx

```diff
@@ -0,0 +1,41 @@
+import { TreeViewBaseItem } from '@mui/x-tree-view';
+
+const useParentUncheckHandler = (
+  groups: TreeViewBaseItem[],
+  selected: string[]
+) => {
+  const findParentIdByItem = (
+    dataSource,
```

Comment:
@FussuChalice: it seems like the type is not set for this one. Could you please add it? Thanks.